chia/rpc/wallet_rpc_api.py

```diff
@@ -1344,14 +1344,15 @@ async def create_new_dl(self, request) -> Dict:
                 )
 
         try:
```

Comment:
Shouldn't there be some transaction handling with commit and rollback to go along with the lock?  I see that is an existing issue and could be addressed separately.